### PR TITLE
Afahs/onm integration

### PR DESCRIPTION
**Pull request by alijawadfahs**

### CODE REVIEW

File: sal-common/src/main/java/org/ow2/proactive/sal/model/Cluster.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 package org.ow2.proactive.sal.model;
 
 import java.util.List;
+import java.util.Map;
```
Comment (AlexFalessi) [inline sal-common/src/main/java/org/ow2/proactive/sal/model/Cluster.java]: No need to import Map

File: sal-service/src/main/java/org/ow2/proactive/sal/service/util/ClusterUtils.java [hunk 2]
```diff
@@ -61,7 +61,7 @@ public class ClusterUtils {
     private RepositoryService repositoryService;
```
Comment (AlexFalessi) [inline sal-service/src/main/java/org/ow2/proactive/sal/service/util/ClusterUtils.java]: I know it's not from your PR but please remove this Autowired field if it's not used.

**Review by AlexFalessi — APPROVED** [review]
Small non blocking comments.

**Review by ankicabarisic — COMMENTED** [review]
Can you please provide the docker image/s to which this code is related to.

**Review by ankicabarisic — APPROVED** [review]

**Review by mboussaa — APPROVED** [review]

File: sal-common/src/main/java/org/ow2/proactive/sal/model/Cluster.java (hunk 1 above)
Comment (alijawadfahs) [reply sal-common/src/main/java/org/ow2/proactive/sal/model/Cluster.java]: done

File: sal-service/src/main/java/org/ow2/proactive/sal/service/util/ClusterUtils.java (hunk 2 above)
Comment (alijawadfahs) [reply sal-service/src/main/java/org/ow2/proactive/sal/service/util/ClusterUtils.java]: done

**Comment by alijawadfahs** [comment]
> Can you please provide the docker image/s to which this code is related to.

alijawad/sal:cluster-draft-11

**Pull request merged**